prototypes/ItemRepeaterPrototypes/ItemsRepeaterExperiments/AttachedBehaviors/SelectionModelPage.xaml.cs

```diff
@@ -0,0 +1,92 @@
+﻿using ItemsRepeaterExperiments.Common;
+using Microsoft.UI.Xaml.Controls;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices.WindowsRuntime;
+using Windows.Foundation;
+using Windows.Foundation.Collections;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Data;
+using Windows.UI.Xaml.Input;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Navigation;
+
+// The Blank Page item template is documented at https://go.microsoft.com/fwlink/?LinkId=234238
+
+namespace ItemsRepeaterExperiments.AttachedBehaviors
+{
+    /// <summary>
+    /// An empty page that can be used on its own or navigated to within a Frame.
+    /// </summary>
+    public sealed partial class SelectionModelPage : Page
+    {
+        private List<Team> teams = DataSourceCreator<Team>.CreateRandomizedList(100);
+        public SelectionModelPage()
+        {
+            this.InitializeComponent();
+            PointerBehaviors.Click += PointerBehaviors_Click;
```

Comment:
Hmm.. This is a static event and not specific to this page, so if we have multiple pages, this event will be raised for all the pages ?

Reply:
Yes unfortunately it would. The problem is that without an instance of the class we can't consolidate the events. I am not sure if you can attach an event as a DP and it would work the way one expect it to.